experiment/run_experiment.py

```diff
@@ -380,9 +380,12 @@ def start(self):
             docker_image_url,
             '/bin/bash',
             '-c',
-            'gsutil -m rsync -r '
-            '"${CLOUD_EXPERIMENT_BUCKET}/${EXPERIMENT}/input" ${WORK} && '
-            'source "/work/.venv/bin/activate" && '
+            'apt-get update && apt-get install -y rsync && '
```

Comment:
If you install rsync in `base-image` and then run `make dispatcher-image` we won't need to do this here.